spells/Increasedlogintime.js

```diff
@@ -0,0 +1,51 @@
+function IncreasedlogintimeSpell(context, payload) {
+	// etat du jeu
```

Comment:
I guess it makes sense to write the comments in french, but since we're working as a team we need to have them in english so everyone understands.